GPUQREngine/Include/GPUQREngine_LLBundle.hpp

```diff
@@ -93,10 +117,14 @@ class LLBundle
     bool Advance();
 
     void PipelinedRearrange();
+
     void UpdateSecondMinIndex();
     void UpdateMax();
+    
 
     void gpuPack(TaskDescriptor *cpuTask);
 };
 
+extern template class LLBundle<int32_t>;
+extern template class LLBundle<int64_t>;
```

Comment:
I'm not exactly sure if these `extern template` declarations will suffice to prevent implicit template instantiations from happening.
Some of the template class functions have arguments of another template class type (`BucketList<Int>`).
To be on the safe side, it might be better to also add `extern template` declarations for the two types of `BucketList` here.